<launch>
<rosparam command="load" file="$(find parsian_world_model)/param/field.yaml" />

<node name="commonconfig" 	pkg="parsian_agent"            	type="parsian_agent_commonconfig_node"  respawn="true" />

<node name="agent_0" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_1" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_2" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_3" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_4" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_5" 		pkg="parsian_agent"           	type="parsian_agent_node"  respawn="true" />


<node name="planner_0" 		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_1" 		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_2" 		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_3" 		pkg="parsian_agent"            	type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_4" 		pkg="parsian_agent"            	type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_5" 		pkg="parsian_agent"            	type="parsian_agent_planner_node"  respawn="true" />

<node name="agent_6" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_7" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_8" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_9" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_10" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />
<node name="agent_11" 		pkg="parsian_agent"            	type="parsian_agent_node"  respawn="true" />


<node name="planner_6"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_7"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_8"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_9"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_10"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />
<node name="planner_11"		pkg="parsian_agent"		type="parsian_agent_planner_node"  respawn="true" />

<node name="planserver" 	pkg="parsian_ai"		type="planserver_node.py"	respawn="true" />

<node name="worldmodel" 	pkg="parsian_world_model"      	type="parsian_world_model_node"	respawn="true" />
<node name="vision"		pkg="parsian_protobuf_wrapper" 	type="vision"	respawn="true" />
<node name="refbox" 		pkg="parsian_protobuf_wrapper" 	type="refbox"	respawn="true" />
<node name="communication_node"	pkg="parsian_communication"	type="com"	respawn="true" />
<node name="packet"		pkg="parsian_communication"	type="packet"	respawn="true" />
<node name="ai_node"		pkg="parsian_ai"		type="parsian_ai_node"	resoawn="true" />
</launch>
